BitwardenShared/UI/Vault/VaultItem/AuthenticatorKeyCapture/ManualEntryView.swift

```diff
@@ -20,27 +20,30 @@ struct ManualEntryView: View {
                 cancelToolbarItem {
                     store.send(.dismissPressed)
                 }
-            }
-    }
 
-    /// A button to trigger an `.addPressed(:)` action.
-    ///
-    private var addButton: some View {
-        Button(Localizations.addTotp) {
-            store.send(
-                ManualEntryAction.addPressed(code: store.state.authenticatorKey)
-            )
-        }
-        .buttonStyle(.secondary())
-        .accessibilityIdentifier("AddTOTPManuallyButton")
+                saveToolbarItem {
+                    store.send(
+                        ManualEntryAction.addPressed(code: store.state.authenticatorKey)
+                    )
+                }
+            }
     }
 
     /// The main content of the view.
     ///
     private var content: some View {
-        VStack(alignment: .leading, spacing: 16.0) {
-            Text(Localizations.enterKeyManually)
-                .styleGuide(.title2, weight: .bold)
+        VStack(alignment: .center, spacing: 24) {
+            VStack(alignment: .center, spacing: 12) {
+                Text(Localizations.enterKeyManually)
+                    .styleGuide(.title2, weight: .semibold)
+
+                Text(Localizations.onceTheKeyIsSuccessfullyEntered)
```

Comment:
:warning: This localization needs to be updated as it indicates "Add TOTP" should be pressed instead of "Save".